packages/react-hooks/src/merkle/useMerkleGroups.ts

```diff
@@ -0,0 +1,32 @@
+import { useCall } from '@webb-dapp/react-hooks';
+import { GroupTree } from '@webb-tools/types/interfaces';
+import { useMemo } from 'react';
+import { hexToU8a } from '@polkadot/util';
+
+class MerkleGroupWrapper {
```

Comment:
I suggest we stick with the same name + `Wrapper` so this should be just `GroupTreeWrapper` so we don't make any other naming issue that we are doing this huge refactor to solve it :D
```suggestion
class GroupTreeWrapper {
```

Reply:
There is a Merkle group and mixer group, this naming follows the query path, not the type name

Reply:
I would suggest following the Type + Wrapper pattern not the query path pattern since that may update.